test/app/e2e/call.test.ts

```diff
@@ -47,19 +48,49 @@ describe('call', () => {
   }
 
   if (device.getPlatform() === 'android') {
-    it('should make an outgoing call and then disconnect', async () => {
-      await element(by.text('CONNECT')).tap();
-      await waitFor(element(by.text('Call State: connected')))
-        .toBeVisible()
-        .withTimeout(DEFAULT_TIMEOUT);
-
-      // Let the call go for 5 seconds
-      await new Promise((r) => setTimeout(r, 5000));
+    describe('outgoing call', () => {
+      it('should make an outgoing call and then disconnect', async () => {
+        await element(by.text('CONNECT')).tap();
+        await waitFor(element(by.text('Call State: connected')))
+          .toBeVisible()
+          .withTimeout(DEFAULT_TIMEOUT);
+
+        // Let the call go for 5 seconds
+        await new Promise((r) => setTimeout(r, 5000));
+
+        await element(by.text('DISCONNECT')).tap();
+        await waitFor(element(by.text('Call State: disconnected')))
+          .toBeVisible()
+          .withTimeout(DEFAULT_TIMEOUT);
+      });
 
-      await element(by.text('DISCONNECT')).tap();
-      await waitFor(element(by.text('Call State: disconnected')))
-        .toBeVisible()
-        .withTimeout(DEFAULT_TIMEOUT);
+      it('should log a constant-audio-input-level quality warning', async () => {
+        await element(by.text('CONNECT')).tap();
+        await waitFor(element(by.text('Call State: connected')))
+          .toBeVisible()
+          .withTimeout(DEFAULT_TIMEOUT);
+
+        // Let the call go for 15 seconds
```

Comment:
why does this call go for 15sec, but the other goes for 5sec?

Reply:
We need this one to go long enough to issue the call quality warnings. The other one only connects and disconnects.